src/components/Search/index.jsx

```diff
@@ -0,0 +1,89 @@
+import algoliasearch from 'algoliasearch/lite';
+import { default as React, useState, useMemo } from 'react';
+import { InstantSearch } from 'react-instantsearch-dom';
+import {
+  ThemeProvider,
+  makeStyles,
+  Box,
+  List,
+  ClickAwayListener,
+} from '@material-ui/core';
+import SearchBox from './search-box';
+import SearchResult from './search-result';
+
+//{window.addEventListener('resize', handleClose)}
+
+const useStyles = makeStyles({
+  root: {
+    background: 'white',
+  },
+  popover: {
+    background: 'white',
+    position: 'absolute',
+    zIndex: 2,
+  },
+  resultsBorder: {
+    borderRadius: 3,
+    borderColor: '#D3D3D3',
+  },
+  search: {
+    maxHeight: 750,
+    width: 'auto',
+    height: 'auto',
+    boxShadow: 5,
+    overflow: 'auto',
+    overflowX: 'hidden',
+  },
+});
+
+export default function Search({ indices }) {
+  const [query, setQuery] = useState();
+  const searchClient = useMemo(
+    () =>
+      algoliasearch(
+        process.env.GATSBY_ALGOLIA_APP_ID,
+        process.env.GATSBY_ALGOLIA_SEARCH_KEY
+      ),
+    []
+  );
+  const theme = useStyles();
+  const [open, setOpen] = React.useState(false);
+  const handleClick = (event) => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+  return (
+    <ThemeProvider theme={theme.root}>
```

Comment:
I think this wrapper shouldn't be needed since we already apply it globally to the whole app. But, similar to what happened above, it looks like you're actually just trying to apply a CSS background. Maybe just make this `<div className={classes.root}?` instead?— after you rename `theme` to `classes`.